### ISSUE TITLE
RTL support

### ISSUE THREAD

**Issue by mehthabux** [callstack/react-native-pager-view#367]
im facing the gesture issue in RTL mode, we need to swipe in opposite direction
im using this with react native tab view and react navigation

**Comment by aleppos** [comment]
@ferrannp How I can pass RTL props https://github.com/callstack/react-native-pager-view/blob/18b9d77792c672cd30e28c86b1576634a890a1cb/android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt#L219

**Comment by troZee** [comment]
Could you confirm, that pager view should behave in below way: 
https://snack.expo.io/@pagerview/rtl

**Comment by mehthabux** [comment]
@troZee yes exactly

**Comment by troZee** [comment]
@mehthabux I implemented RTL for an Android and I was trying to do the same for an iOS, but I don't know why it does not work. I created a separate project, where I implemented it using below tutorial: 

http://redstonetec.blogspot.com/2013/09/uipageviewcontroller-in-right-to-left.html

And it worked. The problem is with calling `viewControllerBeforeViewController` method. In the separate project, this method was called, when user swipes to the left, but it does not call in the lib. 

You can find an implementation for an Android on this branch `feat/rtl`
I can upload this example, if someone would like to take a look on it.

**Comment by aleppos** [comment]
@troZee As I remember the old version of `"@react-native-community/viewpager": "^4.1.0",` it was working fine.

**Comment by mehthabux** [comment]
thank you @troZee , let me check it 👍

**Comment by troZee** [comment]
> @troZee As I remember the old version of `"@react-native-community/viewpager": "^4.1.0",` it was working fine.

I am confused, bc we haven't supported RTL in this library. Could you provide a code snippet with working RTL ?

**Comment by aleppos** [comment]
@mikeqcp it's only work when set layoutDirection='rtl' but still having the same issue when using the default.

`I18nManager.forceRTL(true);`

**Comment by mikeqcp** [comment]
I've tested on Android, and after setting `I18nManager.forceRTL(true);` , I've tried following options:

`viewPager.layoutDirection = View.LAYOUT_DIRECTION_LOCALE`
`viewPager.layoutDirection = View.LAYOUT_DIRECTION_INHERIT`
`viewPager.layoutDirection = if (TextUtilsCompat.getLayoutDirectionFromLocale(Locale.getDefault()) == ViewCompat.LAYOUT_DIRECTION_LTR) View.LAYOUT_DIRECTION_LTR else View.LAYOUT_DIRECTION_RTL`

All of these result in using LTR layout, despite rest of the app is using RTL layout.

@troZee Any ideas why RTL doesn't get applied?

If deducing layout automatically is problematic, maybe we should simplify the `layoutDirection` prop to only allow `ltr / rtl` values with `ltr` as default?
